src/antsibull/build_ansible_commands.py

```diff
@@ -305,46 +304,20 @@ def show_warnings(result: sh.RunningCommand, **kwargs) -> None:
 
 
 def make_dist(ansible_dir: str, dest_dir: str) -> None:
+    # XXX: build has an API, but it's quite unstable, so we use the cli for now
     show_warnings(
         # pyre-ignore[16], pylint:disable-next=no-member
-        sh.python('setup.py', 'sdist', _cwd=ansible_dir),
+        sh.python('-m', 'build', '-s', '-o', dest_dir, ansible_dir),
```

Comment:
I tihnk the long names are better readable:
```suggestion
        sh.python('-m', 'build', '--sdist', '--outdir', dest_dir, ansible_dir),
```

Reply:
What is the use case for building sdist w/o wheels? Is it use for testing from source somewhere? If yes, check out https://github.com/marketplace/actions/checkout-python-sdist.

Reply:
Ansible before 6 has no wheels (https://github.com/ansible-community/antsibull/blob/main/src/antsibull/data/ansible-setup_py.j2#L10-L151).

Reply:
@felixfontein sounds like the two functions could really be one, then. With a little bool flag for adding an extra arg if `build_sdist_only=True`.

Reply:
That guess that could be done more easily now that the main difference between these functions is gone (the copy/check part). I'm not sure it should be done, though. Since we want to remove support for earlier Ansible versions at some point (#472) it is easier to keep it a separate function, since that can be deleted more easily than removing a parameter from a combined function.